bin/sl-run.js

```diff
@@ -31,6 +31,8 @@ config.sendMetrics();
 
 if(!config.clustered) {
   console.log('supervisor running without clustering (unsupervised)');
+} else {
+  config.sendExpressMetrics();
```

Comment:
You can call this unconditionally, like sendMetrics() and just after it, and *inside* sendExpressMetrics make it a no-op if `! this.clustered`.